docs_src/microservices/device/supported/device-onvif-camera/supplementary-info/ONVIF-protocol.md

```diff
@@ -212,6 +212,9 @@ For the concept of implementation:
 └──────────────┘                    └────────────────────┘               └─────────────────┘
 ```
 
+!!! Warning
+		Both REST and soap commands over can be subject to attacks while in transit. Please take all necessary precautions to protect network traffic.
```

Comment:
Please test the rendering of the docs. This one would not have rendered appropriately. Second line (soft return) must be indented by 4 spaces.

```suggestion
!!! warning
    Both REST and soap commands over can be subject to attacks while in transit. Please take all necessary precautions to protect network traffic.
```

Reply:
also, `soap commands over can be subject` wording needs to be fixed. I think it is missing the words `a network`